Sources/Basic/FixableError.swift

```diff
@@ -0,0 +1,23 @@
+/*
+ This source file is part of the Swift.org open source project
+
+ Copyright 2015 - 2016 Apple Inc. and the Swift project authors
+ Licensed under Apache License v2.0 with Runtime Library Exception
+
+ See http://swift.org/LICENSE.txt for license information
+ See http://swift.org/CONTRIBUTORS.txt for Swift project authors
+*/
+
+public protocol FixableError: CustomStringConvertible {
```

Comment:
I wonder if `Basic` is the best place to put this...Usually for something to be a candidate of `Basic` I imagine if that can be leveraged by any package though I am not sure if thats the correct way.